packages/offix-client/src/OfflineClient.ts

```diff
@@ -1,6 +1,6 @@
 import { ApolloClient } from "apollo-client";
-import { DataSyncConfig } from "./config";
-import { SyncConfig } from "./config/SyncConfig";
+import { OffixClientConfig } from "./config";
+import { OffixDefaultConfig } from "./config/OffixDefaultConfig";
```

Comment:
Finally! Now I won't get confused between the interface and the implementation any more 👍

Reply:
Took we a while to figure out the name but yeah! 😿 So happy that you like it!